lib/src/zap/zap_priv.h

```diff
@@ -308,7 +321,15 @@ typedef zap_err_t (*zap_get_fn_t)(zap_t *pz, zap_log_fn_t log_fn,
  * \returns ERANGE For invalid range access.
  * \returns EACCES For invalid access permission.
  */
-int z_map_access_validate(zap_map_t map, char *p, size_t sz, zap_access_t acc);
+static inline
+int z_map_access_validate(zap_map_t map, char *p, size_t sz, zap_access_t acc)
```

Comment:
This function is done in the transport plugin too right? I'm not sure why it's here.